PowerShellGet/PSModule.psm1

```diff
@@ -2873,18 +3005,65 @@ function Publish-Script
                                -CallerPSCmdlet $PSCmdlet `
                                -ErrorCategory InvalidOperation
                 }
-                elseif(-not $Force -and ($currentPSGetItemInfo.Version -gt $PSScriptInfo.Version))
+
+                $galleryScriptVer,$galleryScriptPrerelease = $galleryScriptVersion -split '-',2
+                $toPublishScriptVer,$toPublishScriptPrerelease = $toPublishScriptVersion -split '-',2
+
+                if ($galleryScriptVer -eq $toPublishScriptVer)
+                {
+                    # Prerelease strings will not both be null, otherwise would have terminated already above

```

Comment:
># Prerelease strings will not both be null, otherwise would have terminated already above [](start = 20, length = 89)

Is this statement true when latest version in the gallery is a prerelease and the version being published is not a prerelease version?